cedar-drt/set_env_vars.sh

```diff
@@ -19,6 +19,10 @@ if ! command -v lean &> /dev/null; then
     return 1
 else
     export LEAN_LIB_DIR=$(lean --print-libdir)
+    if [ -z "$LEAN_LIB_DIR" ]; then
+        echo "lean --print-libdir returned no output"
```

Comment:
```suggestion
        echo "error: `lean --print-libdir` returned no output"
```

Reply:
Oh wait -- these backticks would be interpreted. that's not what we want

Reply:
Oops, true. It's fine to remove them, the important part is to make clear that we ran into an error.